src/bin/tikv-ctl.rs

```diff
@@ -197,20 +197,26 @@ impl MvccDeserializable for Vec<u8> {
     }
 }
 
+#[derive(PartialEq, Debug)]
 pub struct MvccKv<T> {
     key: Key,
     value: T,
 }
 
 pub fn gen_mvcc_iter<T: MvccDeserializable>(db: &DB,
                                             key_prefix: &str,
+                                            prefix_encoded: bool,
```

Comment:
Seem that prefix_encoded is the `encoded` arg in other functions, why use a different name here?